test/models/edition_test.rb

```diff
@@ -267,29 +267,6 @@ def draft_second_edition_from(published_edition)
     assert_equal edition.whole_body, new_edition.whole_body
   end
 
-  test "Cloning from LicenceEdition into AnswerEdition" do
```

Comment:
Hmm, might be worth double-checking if there's code to do this - maybe it's not needed? What did they do with the other retirements, did any of these have AnswerEdition cloning?